### PR TITLE
Push configuration data into recipes

### PR DESCRIPTION
**Pull request by dpquigl2**
Several recipes worth of CFLAGS LDFLAGS and ASNEEDED options were specified in local.conf. This pushes those statements into the recipes themselves

OXT-308

Signed-Off-By: David Quigley <drquigl@tycho.nsa.gov>

### CODE REVIEW

File: recipes-extended/pigz/pigz_2.2.4.bbappend [hunk 1]
```diff
@@ -1,2 +0,0 @@
-SRC_URI = "${DEBIAN_MIRROR}/main/p/pigz/pigz_${PV}.orig.tar.gz \
-           file://ldflags.patch"
```
Comment (cjp256) [inline recipes-extended/pigz/pigz_2.2.4.bbappend]: Is this patch still being applied somewhere?  Should this be a +=?

**Comment by cjp256** [comment]
Missing commit signoffs.  And just the Q about the pigz ldflags patch, but otherwise looks good to me.

File: recipes-extended/pigz/pigz_2.2.4.bbappend (hunk 1 above)
Comment (dpquigl2) [inline recipes-extended/pigz/pigz_2.2.4.bbappend]: I don't think it is. Its not clear to me why it needed at all. It builds just find an works without either the modified source package or the patch.

**Comment by rossphilipson** [comment]
See my comment in https://github.com/OpenXT/openxt/pull/61 about PRs being left open for long periods...

**Comment by dpquigl2** [comment]
Bring picked up by other people to finish.

**Pull request closed without merging**